rocks/jimm.yaml

```diff
@@ -39,7 +39,9 @@ parts:
             # place it in bin
             cp -r ./internal/dbmodel/sql $CRAFT_PART_INSTALL/usr/local/bin/sql
 
-            # Copied for the charm to collect and apply, not this container itself
+            # Convert the OpenFGA auth model to JSON and place a copy in the ROCK
+            # Used by the charm to apply to the OpenFGA server, not by the container itself
             mkdir -p $CRAFT_PART_INSTALL/root/openfga/
-            cp ./openfga/authorisation_model.json $CRAFT_PART_INSTALL/root/openfga/authorisation_model.json
-
+            # Note that we can't use go run directly (https://github.com/canonical/rockcraft/issues/755)
+            go install github.com/openfga/cli/cmd/fga@latest
+            /root/go/bin/fga model transform --file ./openfga/authorisation_model.fga --output-format json > $CRAFT_PART_INSTALL/root/openfga/authorisation_model.json
```

Comment:
why do you need the json file in the first place? The docker compose doesn't use it anymore, and you are transforming the yaml for the backend and openfga entrypoint.
What's missing?

Reply:
The comment above explains why it's in the rock, it's for the charm to retrieve and then set on the OpenFGA server when creating the store.

Reply:
yes, but isn't the openfga charm able to now being started with the yaml, without worrying about the json?
From the docker compose it seems openfga doesn't need the json anymore.

Reply:
In the docker compose it also needs the json.

Reply:
Look at the changes in entrypoint.sh to see where I convert it.

Reply:
ah yes, you are right!